docs/content/community/overview.md

```diff
@@ -25,3 +25,7 @@ If you would like to file Issues, access the source code, or use Codespaces plea
 ## How can I get involved?
 
 One of the easiest ways to contribute is to participate in discussions at community engagements or via the Discord server referenced above. We are always seeking feedback, especially for suggestions around improving the product and docs.
+
+## Artifacts on Radius
```

Comment:
Have we considered adding a top level tab, something like `Learn More` on https://radapp.io/, with a drop down menu for each of the artifact type? Putting it under `community` might make it hard to discover, and it feels like these resources should be quickly accessible when a new user visits the website.

Reply:
Great idea! I can link the same page as a tab from radapp.io. As we grow, we can split them by artifact type.